haskell/test/Lorentz/Contracts/Test/Management.hs

```diff
@@ -610,6 +665,12 @@ managementSpec originate = do
             StorageSafelistContract Nothing -> Right ()
             _ -> Left $ CustomValidationError "Token produced malformed storage, this should not have happened"
 
+      it "validates safelist interface" $ integrationalTestExpectation $ do
```

Comment:
I think we could come up with a better description of this test that at least explains that it should fail due to bad safelist parameter

Reply:
Have expanded it to say "should fail if parameter of safelist contract does not have the required entrypoints"